examples/fill_and_plot_energy_offset_array.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
Could you please also fix the code formatting in the example file?
See [pep8 examples/fill_and_plot_energy_offset_array.py output](https://gist.github.com/cdeil/97993acb03652f03f2a0)

Also: Please add at least a one-line docstring at the top of every new file, in this case something like:
```
"""Examples how to use EnergyOffsetArray.
"""
```
and then this example file can be extended in future PRs when new methods are implemented (e.g. slice out spectra or radial acceptance curves or ...).